Basic FE project template/ui-src/styles/general/global/_global.scss

```diff
@@ -1,3 +1,33 @@
-div {
-  color: red;
+* {
+  margin: 0;
+  top: 0;
+  box-sizing: border-box;
+}
+.container {
+  width: 1380px;
```

Comment:
change witdth to 100%, add max-width: 1380px; and move the margins to the why-beleasing section